unit/models/repayment.py

```diff
@@ -0,0 +1,109 @@
+from typing import Optional, Dict, Union, List, Literal
+
+from unit.models import UnitDTO, extract_attributes, UnitRequest, Relationship, UnitParams
+from unit.utils import date_utils
+
+
+class BaseRepayment(UnitDTO):
+    def __init__(self, _id, _type, attributes, relationships):
+        self.id = _id
+        self.type = _type
+        self.attributes = extract_attributes(["amount", "status", "tags"], attributes)
+        attrs = {"createdAt": date_utils.to_datetime(attributes["createdAt"]),
+                 "updatedAt": date_utils.to_datetime(attributes["updatedAt"])}
+        self.attributes.update(attrs)
+        self.relationships = relationships
+
+
+class BookRepaymentDTO(BaseRepayment):
```

Comment:
Maybe I am wrong but tags are inside of attributes, and I they are missing

Reply:
`self.attributes = extract_attributes(["amount", "status", "tags"], attributes)`